docs/operator_guides/0_running_an_operator.md

```diff
@@ -152,6 +152,65 @@ If you don't have Holesky ETH, these are some useful faucets:
 ./operator/build/aligned-operator start --config ./config-files/config-operator.yaml
 ```
 
+### Run Operator using Systemd
+
+To manage the Operator process on Linux systems, we recommend use Systemd with the following configuration:
+
+```shell
+# aligned-operator.service
+
+[Unit]
+Description=Aligned Operator
+After=network.target
+
+[Service]
+Type=simple
+User=<user>
+WorkingDirectory=<path_to_aligned_layer_repository>
+ExecStart=<path_to_aligned_layer_repository>/operator/build/aligned-operator start --config <path_to_operator_config>
+Restart=always
+RestartSec=1
+StartLimitBurst=100
+
+[Install]
+WantedBy=multi-user.target
+```
+
+{% hint style="info" %}
+`aligned-operator.service` is just an arbitrary name. You can name your service as you wish, following the format `<service-name>.service`.
+{% endhint %}
```

Comment:
This is not rendering correctly on GitHub's UI.

Reply:
This is the supported format for gitbook, but github does not support it. Since this docs are supposed to be watched in docs.alignedlayer.com, we prefer to use the gitbook format

Reply:
We use gitbook to render the docs, and that is the reason why we use that format